frontend/src/components/NotificationsCard.jsx

```
@@ -288,6 +328,53 @@ export default function NotificationsCard() {
+              label="Notify URL String"
+              value={config.notify_url_string || ""}
+              onChange={e => handleChange("notify_url_string", e.target.value)}
+              size="small"
+              multiline={showNotifyString}
+              minRows={showNotifyString ? 2 : undefined}
+              sx={{ minWidth: 350, maxWidth: 600 }}
+              //helperText="Comma-separated Apprise URLs (e.g., tgram://TOKEN/CHATID, pover://USER_KEY@TOKEN discord://WEBHOOK_ID/WEBHOOK_TOKEN)."
+              helperText={
+                <>
+                  Comma-separated Apprise URLs. See the
+                  <a href="https://github.com/caronc/apprise/wiki#notification-services" target="_blank"> Apprise docs</a>.
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)